numpydoc/docscrape.py

```diff
@@ -236,10 +236,30 @@ def _parse_param_list(self, content):
 
         return params
 
+    _role = r":(?P<role>\w+):"
+    _funcbacktick = r"`(?P<name>(?:~\w+\.)?[a-zA-Z0-9_.-]+)`"
+    _funcplain = r"(?P<name2>[a-zA-Z0-9_.-]+)"
+    _funcname = r"(" + _role + _funcbacktick + r"|" + _funcplain + r")"
+    _funcnamenext = _funcname.replace('role', 'rolenext').replace('name', 'namenext')
+    _description = r"(?P<description>\s*:(\s+(?P<desc>\S+.*))?)?\s*$"
+    _func_rgx = re.compile(r"^\s*" + _funcname + r"\s*", re.X)
```

Comment:
why re.X? I don't think you're taking advantage of verbose here anywhere (if you were, you might not need to define all these intermediate variables as you could put comments in the regex)

Reply:
`re.X` was leftover from original `_name_rgx`.  Now removed.